tests/test_playbooks/content_export_repository.yml

```diff
@@ -0,0 +1,70 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include_tasks: tasks/organization.yml
+      vars:
+        organization_state: present
+        organization_name: "Export Org"
+    - include_tasks: tasks/product.yml
+      vars:
+        product_name: "Export Test Product"
+        organization_name: "Export Org"
+        product_state: present
+    - include_tasks: tasks/repository.yml
+      vars:
+        product_name: "Export Test Product"
+        repository_name: "TestRepo"
+        repository_state: present
+        organization_name: "Export Org"
+      register: result
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - include_tasks: tasks/repository.yml
+      vars:
+        product_name: "Export Test Product"
+        repository_name: "TestRepo"
+        repository_state: present
+        organization_name: "Export Org"
+      register: result
+    - include_tasks: tasks/content_export_repository.yml # Full export
+      vars:
+        product: "Export Test Product"
+        repository: "TestRepo"
+        expected_change: true
+        incremental: false
+        organization_name: "Export Org"
+        foreman_organization: "Export Org"
+    - include_tasks: tasks/repository.yml # i dont know why but it loses result in the next step if I don't include repository.yml here
+      vars:
+        product_name: "Export Test Product"
+        repository_name: "TestRepo"
+        repository_state: present
+        organization_name: "Export Org"
+      register: result
+    - include_tasks: tasks/content_export_repository.yml # Incremental export
+      vars:
+        product: "Export Test Product"
+        repository: "TestRepo"
+        expected_change: true
+        incremental: true
+        organization_name: "Export Org"
+        foreman_organization: "Export Org"
+    - include_tasks: tasks/content_export_show.yml
+      vars:
+        id: "{{ result['task']['output']['export_history_id'] | default(12345) }}"
+        organization_name: "Export Org"
+        foreman_organization: "Export Org"
+        destination_server: "airgapped.example.com"
+      when:
+        - not ansible_check_mode
+...
```

Comment:
can you add another play, that goes against `localhost` again and tears down the created repo/product/org?